code/__HELPERS/stack_trace.dm

```diff
@@ -0,0 +1,11 @@
+///gives us the stack trace from CRASH() without ending the current proc.
+/proc/stack_trace(msg)
+	CRASH(msg)
+
+GLOBAL_REAL_VAR(list/stack_trace_storage)
```

Comment:
Use a static var

Reply:
tried to make a `var/static/list/stack_trace_storage = list()` in the proc, but when building it errors because this list is used in other places too, what do i do?

Reply:
Ugh, it's used in the runtime handler. Just nevermind then.